tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "noImplicitAny": false,
-    "noUnusedLocals": true,
+    // "noUnusedLocals": true,
```

Comment:
If you can find a way to make this throw a warning but not fail I'm all for removing it